### PR TITLE
En 3563 update term ui

### PR DESCRIPTION
**Pull request by iulianpascalau**
updated gin to start in debug mode only if termui is not enabled
code refactoring in termui render
added version label in termui render

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ type prometheus struct {
 // MainApiHandler interface defines methods that can be used from `elrondFacade` context variable
 type MainApiHandler interface {
 	RestApiPort() string
+	GinDebugMode() bool
```
Comment (camilbancioiu) [inline api/api.go]: Could rename `Gin` to `RestAPIServer`.

File: cmd/node/main.go [hunk 2]
```diff
@@ -642,7 +643,7 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		return err
 	}
 
-	ef := facade.NewElrondNodeFacade(currentNode, apiResolver)
+	ef := facade.NewElrondNodeFacade(currentNode, apiResolver, !useTermui)
```
Comment (camilbancioiu) [inline cmd/node/main.go]: For readability, you could write above: `ginDebugMode := !useTermui` then pass it as the argument to `NewElrondNodeFacade`.

File: statusHandler/termuiStatusHandler_test.go [hunk 3]
```diff
@@ -25,25 +25,14 @@ func TestTermuiStatusHandler_TermuiShouldPass(t *testing.T) {
...
-}
-
 func TestTermuiStatusHandler_TestIncrement(t *testing.T) {
 	t.Parallel()
 
 	var metricKey = core.MetricNonce
 
 	termuiStatusHandler := statusHandler.NewTermuiStatusHandler()
 
+	termuiStatusHandler.SetUInt64Value(metricKey, 0)
```
Comment (camilbancioiu) [inline statusHandler/termuiStatusHandler_test.go]: For readability, delete `var metricKey` and use `core.MetricNonce` directly as the arguments to `SetUInt64Value` here.

File: api/api.go (hunk 1 above)
Comment (iulianpascalau) [reply api/api.go]: done

File: cmd/node/main.go (hunk 2 above)
Comment (iulianpascalau) [reply cmd/node/main.go]: ok

Comment (iulianpascalau) [reply cmd/node/main.go]: done

File: statusHandler/termuiStatusHandler_test.go (hunk 3 above)
Comment (iulianpascalau) [reply statusHandler/termuiStatusHandler_test.go]: done

**Pull request merged**